config/jobs/kubernetes/node-problem-detector/node-problem-detector-ci.yaml

```diff
@@ -56,6 +56,10 @@ periodics:
     repo: node-problem-detector
     base_ref: master
     path_alias: k8s.io/node-problem-detector
+  - org: kubernetes
```

Comment:
@krzyzacy, for the extra_refs, the default path is in the first repository, right?

Reply:
yes